hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -671,7 +668,6 @@ instance ToErrorCode HeadError where
     MissingCommits -> "H08"
     HeadValueIsNotPreserved -> "H09"
     HasBoundedValidityCheckFailed -> "H10"
-    InvalidSnapshotSignature -> "H11"
```

Comment:
This is exactly why we would need the template haskell replacing, so we can keep the numbers not scattered around. Or maybe, numbering was a bad idea from me.

Reply:
Yeah, I know but, do want me to fix the numbering in this PR?

I left this behind for that reason, thinking it would be better to fix it as part of the PR to add TH.

wdyt?

Reply:
Yes, that's okay